packages/dht/test/kademlia-simulation/DhtNode.ts

```diff
@@ -0,0 +1,123 @@
+import KBucket from 'k-bucket'
+import { Contact } from './Contact'
+import { SortedContactList } from './SortedContactList'
+
+export class DhtNode {
+
+    private numberOfNodesPerKBucket = 1
+    private K = 8
```

Comment:
Constants

Reply:
Add `readonly` modifier

Reply:
COnstants are now in DhtNodeConfig class